cli/cmd/generate_aws.go

```diff
@@ -494,7 +546,63 @@ func validateAwsProfile(val interface{}) error {
 	return validateStringWithRegex(val, fmt.Sprintf(`^%s$`, AwsProfileRegex), "invalid profile name supplied")
 }
 
-func promptAgentlessQuestions(config *aws.GenerateAwsTfConfigurationArgs) error {
+func promptAgentlessQuestions(
+	config *aws.GenerateAwsTfConfigurationArgs,
+	extraState *AwsGenerateCommandExtraState,
+) error {
+	if err := SurveyMultipleQuestionWithValidation([]SurveyQuestionWithValidationArgs{
+		{
+			Prompt: &survey.Confirm{
+				Message: QuestionEnableAgentlessOrganization,
+				Default: config.AwsOrganization,
+			},
+			Response: &config.AwsOrganization,
```

Comment:
Should there be a check on this? If the flag has been passed should we omit this question?

Reply:
There are same problems for the other flags. Will handle it in the refactoring work.